tests/Query/Mysql/DateTest.php

```diff
@@ -13,6 +13,15 @@ public function testDateDiff()
         $this->assertEquals($sql, $q->getSql());
     }
 
+    public function testFrom_Unixtime()
+    {
+        $dql = "SELECT p FROM DoctrineExtensions\Tests\Entities\Date p WHERE FROM_UNIXTIME(CURRENT_TIME())";
+        $q = $this->entityManager->createQuery($dql);
+
+       $sql = "SELECT d0_.id AS id0, d0_.created AS created1 FROM Date d0_ WHERE FROM_UNIXTIME(CURRENT_TIME)";
```

Comment:
I think if you just stick `()` on the end of `CURRENT_TIME` in on line 21 you should be good to go.